# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/GfxInfo.cpp
# Commit: 3073dc4d247d
# Full Hash: 3073dc4d247d511cac549b93e2c86cd585f64489
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2023-03-08 09:48:25
# Regressor Bug: 1819466
# File Overlap Count: 1
# Description:
#   Bug 1820267 - Unship EGL on Linux for older NVIDIA binary drivers. r=gfx-reviewers,lsalzman
#   
#   After landing bug 1819466, we started seeing a consistent uptick in
#   crashes on older drivers using WebGL. Given the limited population on
#   older drivers on nightly, this is a sign we are unable to ship EGL on
# ==============================================================================

diff -r cba8b9fb276c -r 3073dc4d247d widget/gtk/GfxInfo.cpp
--- a/widget/gtk/GfxInfo.cpp	Tue Mar 07 22:43:20 2023 +0000
+++ b/widget/gtk/GfxInfo.cpp	Tue Mar 07 23:05:04 2023 +0000
@@ -729,21 +729,12 @@
         DRIVER_LESS_THAN, V(18, 2, 0, 0), "FEATURE_X11_EGL_OLD_MESA_NOUVEAU",
         "Mesa 18.2.0.0");
 
-#ifdef EARLY_BETA_OR_EARLIER
-    APPEND_TO_DRIVER_BLOCKLIST_EXT(
-        OperatingSystem::Linux, ScreenSizeStatus::All, BatteryStatus::All,
-        WindowProtocol::All, DriverVendor::NonMesaAll, DeviceFamily::NvidiaAll,
-        nsIGfxInfo::FEATURE_X11_EGL, nsIGfxInfo::FEATURE_BLOCKED_DRIVER_VERSION,
-        DRIVER_LESS_THAN, V(340, 108, 0, 0),
-        "FEATURE_ROLLOUT_X11_EGL_NVIDIA_BINARY_EARLY_BETA", "340.108.0");
-#else
     APPEND_TO_DRIVER_BLOCKLIST_EXT(
         OperatingSystem::Linux, ScreenSizeStatus::All, BatteryStatus::All,
         WindowProtocol::All, DriverVendor::NonMesaAll, DeviceFamily::NvidiaAll,
         nsIGfxInfo::FEATURE_X11_EGL, nsIGfxInfo::FEATURE_BLOCKED_DRIVER_VERSION,
         DRIVER_LESS_THAN, V(470, 82, 0, 0),
-        "FEATURE_ROLLOUT_X11_EGL_NVIDIA_BINARY_RELEASE", "470.82.0");
-#endif
+        "FEATURE_ROLLOUT_X11_EGL_NVIDIA_BINARY", "470.82.0");
 
     // Disable on all AMD devices not using Mesa.
     APPEND_TO_DRIVER_BLOCKLIST_EXT(
